### PR TITLE
Switch to using jupyterlite-xeus instead of xeus-python

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

**Comment by martinRenou** [comment]
Check-release deciding again to use system yarn 😠

**Comment by trungleduc** [comment]
you may need to activate the  conda environment with `init-shell: bash` in the `mamba-org/setup-micromamba@v1` action and activate the bash shell in the steps with `shell: bash -l {0}`

**Comment by martinRenou** [comment]
Thanks! I was sure I did that already in the last PR.

**Comment by martinRenou** [comment]
@trungleduc it does not work

Actually I looked into doing it at some point but in the end I mimicked what we do in jupytercad, but it does not seem to work here??

**Comment by martinRenou** [comment]
Check-release now works nicely. I'll merge and we can iterate.

Next steps:

- add some documentation
- use the mounts feature for jupytelite-xeus seamlessly

File: .github/workflows/check-release.yml [hunk 1]
```diff
@@ -21,16 +16,19 @@ jobs:
...
+          environment-name: voici-dev
           create-args: >-
             python=3.9
             yarn=3
+            pip
+            jupyter-releaser
 
       - name: Check Release
-        uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v2
+        uses: jupyter-server/jupyter_releaser/.github/actions/check-release@main
```
Comment (jtpio) [inline .github/workflows/check-release.yml:27]: Looks like https://github.com/jupyter-server/jupyter_releaser/pull/549 was released as part of https://github.com/jupyter-server/jupyter_releaser/releases/tag/v1.6.0, so we can stick to `v2`.

Comment (martinRenou) [reply .github/workflows/check-release.yml:27]: Right! Thank you I'll make the change in the coming PR

**Comment by jtpio** [comment]
> use the mounts feature for jupytelite-xeus seamlessly

According to the discussion in https://github.com/voila-dashboards/voici/issues/104#issuecomment-1895443970, it looks like this is likely to be implemented in the xeus addon in https://github.com/jupyterlite/xeus directly?

**Pull request merged**